### PR TITLE
tests: add test cases for FTP logging

### PR DESCRIPTION
**Pull request by jlucovsky**
This PR adds tests for FTP logging. See (3093)[https://github.com/OISF/suricata/pull/3093]

### CODE REVIEW

File: tests/output-eve-ftp/suricata.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+%YAML 1.1
```
Comment (victorjulien) [inline tests/output-eve-ftp/suricata.yaml]: why is this needed? The default config has ftp enabled.

File: tests/output-eve-ftp/test.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
```
Comment (victorjulien) [inline tests/output-eve-ftp/test.yaml]: we will need a check for the version as well, as 4.1.x will fail this test

File: tests/output-eve-ftp/suricata.yaml (hunk 1 above)
Comment (jlucovsky) [reply tests/output-eve-ftp/suricata.yaml]: Will remove.

File: tests/output-eve-ftp/test.yaml (hunk 2 above)
Comment (jlucovsky) [reply tests/output-eve-ftp/test.yaml]: Will add a `files` requirement for `src/output-json-ftp.c` which was added for FTP logging.

**Comment by jlucovsky** [comment]
Continued in #106
